datapackage_pipelines_measure/processors/add_github_resource.py

```diff
@@ -0,0 +1,55 @@
+import collections
+import tempfile
+import json
+import itertools
+
+import requests
+import tabulator
+from datapackage_pipelines.wrapper import ingest, spew
+
+from datapackage_pipelines_measure.config import settings
+
+import logging
+log = logging.getLogger(__name__)
+
+parameters, datapackage, res_iter = ingest()
+
+name = str(parameters['name'])
+repo = parameters.get('repo')
+repo_url = '{}{}?access_token={}'.format(settings.GITHUB_API_BASE_URL,
+                                         repo,
+                                         settings.GITHUB_API_TOKEN)
+
+try:
+    repo_content = requests.get(repo_url).json()
+except json.decoder.JSONDecodeError:
+    log.error('Expected JSON in response from: {}'.format(repo_url))
+    raise
+
+# remap retrieved dict to scheme in parameters
+resource_content = {t_key: repo_content[s_key]
+                    for t_key, s_key in parameters['map_fields'].items()}
+
+resource_content = collections.OrderedDict(sorted(resource_content.items(),
+                                                  key=lambda t: t[0]))
+
+resource = {
+    'name': name,
+    'path': 'data/{}.json'.format(name)
+}
+
+with tempfile.NamedTemporaryFile(suffix='.json', delete=False) as out:
+    out.write(bytes(json.dumps([resource_content]), encoding='utf-8'))
+
+headers = list(resource_content.keys())
+
+with tabulator.Stream('file://'+out.name,
```

Comment:
Fixed this up so it doesn't need tabulator or the temp file. I couldn't pass `[resource_content]` in directly without first making it a generator that yields `[resource_content]`. Otherwise it just iterates over the keys. Figured it out in the end :)